package.json

```diff
@@ -39,6 +39,7 @@
     "bootstrap": "^4.2.1",
     "camelcase-keys": "^5.0.0",
     "classnames": "^2.2.5",
+    "connected-react-router": "^5.0.1",
```

Comment:
As said in the summary, changing the router and upgrading the react/redux packages wasn't strictly necessary, but I didn't see any reason to waste the work.